MultimodalQnA/ui/gradio/multimodalqna_ui_gradio.py

```diff
@@ -431,6 +431,31 @@ def clear_text(request: gr.Request):
     return None
 
 
+def get_files():
+    try:
+        response = requests.post(dataprep_get_file_addr, headers=headers)
+        logger.info(response.status_code)
+        files = response.json()
```

Comment:
If no files are returned, maybe display a message about there being no files in the vector store.